kratos/python_scripts/eigen_solver_factory.py

```diff
@@ -22,27 +22,27 @@ def ConstructSolver(settings):
     elif solver_type == "eigen_feast":
         if kratos_utils.CheckIfApplicationsAvailable("EigenSolversApplication"):
             import KratosMultiphysics.EigenSolversApplication as EiSA
-            try:
+            if EiSA.KratosEigenSolversApplication().HasFEAST():
```

Comment:
```suggestion
            if EiSA.HasFEAST():
```

doesn't this work?

Reply:
no. EiSA is the module and I expose `HasFEAST()` to the class `KratosEigenSolversApplication`

Reply:
ah ok I see

Can you please make `HasMKL` and `HasFEAST` `static` such that you don't have to create an Application on the fly?

Reply:
I managed to define it for the module. Do you like it?